src/python_bring_api/bring.py

```diff
@@ -73,8 +73,16 @@ def login(self) -> BringAuthResponse:
             r = requests.post(f'{self.url}bringauth', data=data)
             r.raise_for_status()
         except RequestException as e:
-            _LOGGER.error(f'Exception: Cannot login:\n{traceback.format_exc()}')
-            raise BringRequestException(f'Authentication failed due to request exception.') from e
+            if e.response.status_code == 401: 
+                errmsg = e.response.json()

```

Comment:
On line 88, we check whether the json can be parsed. If it cannot be parsed here, we do not catch it. If we want to use the errmsg of the response sent by the api, we need also to check it here.

Reply:
Would this change be ok?

```
if e.response.status_code == 401:
    try:
        errmsg = e.response.json()
    except JSONDecodeError:
        _LOGGER.error(f'Exception: Cannot parse login request response:\n{traceback.format_exc()}')
    else:
        _LOGGER.error(f'Exception: Cannot login: {errmsg['message']}') 
    raise BringAuthException('Login failed due to authorization failure, please check your email and password.') from e
```
 or should a BringParseException be raised instead of a BringAuthException?

Reply:
I would say the http error takes prio, but the json error is still worth to be logged.